### ISSUE TITLE
[Bug]: PC0031 Per documentation, Modify does not cause JIT loads

### ISSUE THREAD

**Issue by StefanMaron** [ALCops/Analyzers#265]
### Analyzer

PlatformCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

PC0031

### Description

Per documentation, Modify does not cause JIT loads

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-partial-records

`For performance reasons, it's not recommended to use partial records on a record that will do inserts, deletes, renames, field transfers, or copies to temporary records. All these operations require that all fields on the record are loaded, so the platform will emit a JIT load if they're not already loaded.`

### AL code to reproduce

```al

```

**Comment by Arthurvdv** [comment]
Thanks for pointing me to this. I somehow included all database actions, where the documentation clearly states the `inserts, deletes, renames` operators.

I've excluded the `Modify` from the diagnostic with the release of version `v0.8.1` of ALCops.
